roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,35 @@
   with_items: network_interfaces_d
   tags: ['prereboot']
 
+- name: set slrouter address
+  set_fact: sl_router_address="{{ ansible_default_ipv4.network.split('.')[0] }}.{{ ansible_default_ipv4.network.split('.')[1] }}.{{ansible_default_ipv4.network.split('.')[2] }}.{{ (ansible_default_ipv4.network.split('.')[3] | int) + 1 }}"
+  when: ansible_default_ipv4.interface == hostvars[inventory_hostname][primary_interface].device   
+
+- name: add static route for multi subnet
+  add_sys_route: state=present
```

Comment:
I was unaware of `ip route replace`. I'd rather see that used than a whole new module, until we are certain that the `ip route replace` facility is insufficient.

I'm not super keen about there always being work done and a "change" made on each run, particularly around routing, but if that's just the way its done, I can accept that.